fhttp/http_client.go

```diff
@@ -311,6 +310,14 @@ func (c *Client) Close() int {
 	return 0 // TODO: find a way to track std client socket usage.
 }
 
+// GetRequestSize returns the message size
+func (c *Client) GetRequestSize() int {
+	if c.reqDump == nil {
```

Comment:
not needed
https://golang.org/pkg/builtin/#len

Reply:
My purpose was to get request size from one method for two different structs otherwise I need to cast the interface to `FastClient` or `Client` to get request size. Whats your recommendation here? Should I cast the interface to `FastClient` or `Client` to get request size in `Run` method?

Reply:
I am just pointing out that you don't need to test if reqDump is `nil` you can replace by just line 318

`return len(c.reqDump)`